README.md

```diff
@@ -1,2 +1,73 @@
-# baby-bliss-bot
-An exploratory research to generate new Bliss vocabulary
+# Baby Bliss Bot
+
+An exploratory research project to generate new Bliss vocabulary using machine learning techniques.
+
+[The Bliss language](https://www.blissymbolics.org/) is an Augmentative and Alternative Communication (AAC) language
+used by individuals with severe speech and physical impairments around the world, but also by others for language
+learning and support, or just for the fascination and joy of this unique language representation. It is a semantic
+graphical language that is currently composed of more than 5000 authorized symbols - Bliss-characters and Bliss-words.
+It is a generative language that allows its users to create new Bliss-words as needed.
+
+We are exploring the generation of new Bliss vocabulary using emerging AI techniques, including Large Language Models
+(LLM), OCR, Stable Diffusion and other models for text generation and completion.
+
+## Local Installation
+
+### Prerequisites
+
+* [Python 3](https://www.python.org/downloads/)
+  * Version 3.9+. On Mac, Homebrew is the easiest way to install.
+* [virtualenv](https://virtualenv.pypa.io/en/latest/) 
+  * Not required, but highly recommended for maintaining an isolated environment for Python and Python packages.
```

Comment:
I've had problems with `vitualenv` on macOS, and switched to using [`conda`](https://whiteboxml.com/blog/the-definitive-guide-to-python-virtual-environments-with-conda) instead. 

However, it's not clear that anything as heavyweight as `conda` is needed at the present time.  The instructions for virtual environments in the "Create/Activate Virtual Environment" section below is sufficient, using the built-in python virtual environment setup.

If we end up using PyTorch at some point, then a more sophisticated virtual environment/package manager like conda would be useful.  I suggest removing the advice to use `virtualenv` here.

Reply:
This branch needs to install a couple of python packages. Personally I think it's a good practice to use `virtualenv` whenever extra python packages need to be installed.

Reply:
I'm not arguing against using virtual environments.  I'm saying that the built-in `venv` module is sufficient for now.

This README has instructions for using `venv` to create and activate a virtual environment in the "Create/Activate Virtual Environment" section.  But it does not say when or how to use `virtualenv`.

Also, I confused `virtualenv` with `pyenv`, another virtual environment manager. It was `pyenv` that gave me grief, so I abandoned it in favour of `conda`.  So, my negative comments about `virtualenv` are misplaced. I'm actually neutral towards it.  But, I don't see where `virtualenv` is used in this README.

Reply:
You are right, @klown. `virtualenv` and `venv` are different concepts. `venv` is a subset of `virtualenv`. It was built into the python since the version 3.3. `virtualenv` should not be one of the system requirements. I've remove it from the Prerequisites section. Your comment helps me to clarify the understanding. Thanks.